crates/server/src/main.rs

```diff
@@ -145,10 +168,37 @@ async fn try_handle_connection(conn: ft::File) -> ft::Result<()> {
     client.send_raw(b"BIENVENUE\n").await?;
     let team_name = client.recv_line().await?;
 
-    ft_log::trace!(
-        "received team name: `{}`",
-        core::str::from_utf8(team_name).unwrap_or("<invalid-utf8>")
-    );
+    if team_name == b"GRAPHIC" {
+        ft_log::trace!("client #{id} is a graphical monitor");
+        todo!();
```

Comment:
Usually todo's are supposed to be things that are to be changed before a commit, so they are not supposed to be pushed. For unimplemented code use instead `unimplemented!()'

Reply:
trueeeee